git-cliff/src/lib.rs

```diff
@@ -114,6 +114,33 @@ pub fn run(mut args: Opt) -> Result<()> {
 	// Parse tags.
 	let mut tags = repository.tags(&config.git.tag_pattern, args.topo_order)?;
 
+	// Skip tags.
+	let skip_regex = config.git.skip_tags.as_ref();
+	let ignore_regex = config.git.ignore_tags.as_ref();
+	tags = tags
+		.into_iter()
+		.filter(|(_, name)| {
+			// Keep skip tags to drop commits in the later stage.
+			let skip = skip_regex.map(|r| r.is_match(name)).unwrap_or_default();
```

Comment:
Can you elaborate this part a little bit? Why `skip_regex` is needed here and doesn't it cause a duplicate behaviour with this:

https://github.com/orhun/git-cliff/blob/ef63727b5f0a5aba7024e2afe5dc24a1b218d978/git-cliff/src/changelog.rs#L91-L97

Reply:
Suppose that `skip_tags = "v0.1.0-beta.1"` and `ignore_tags = "v.*-beta.*"`.
Then, if `v0.1.0-beta.1` is removed here, it cannot skip(drop) commits in `changelog.rs` because only the tags in `tags` variable are processed.

Please see the following results.

### With this part

```sh-session
$ cargo run -- --config .github/test-git-cliff/cliff.toml | tail -n 40
### Refactor

- Make tag_pattern optional

## [0.1.0-rc.1] - 2021-06-16

### Bug Fixes

- Install musl-tools for musl builds
- Include configuration file in the binary releases
- Specify the bash as shell
- Use bash while setting the release version

### Documentation

- Update the doc comment for completions script
- Add usage section

### Features

- Add `--workdir` argument
- Show the processsed commit message

### Miscellaneous Tasks

- Update config
- Set the release body text
- Add docker releases
- Add release script
- Update the release script about arguments
- Strip the markdown format from tag message

### Refactor

- Update the value name for `--strip`
- Improve logging
- Update value names and description

<!-- generated by git-cliff -->
```

### Without this part

```sh-session
$ # commits in v0.1.0-beta.1 will appear
$ cargo run -- --config .github/test-git-cliff/cliff.toml | tail -n 40
- Update `Release` struct
- Remove unreleased_title entry from config file
- Use `Release` type as changelog context
- Use template from config file
- Use git-conventional crate from crates.io
- Rename tag_regex to tag_pattern
- Use timestamp for release dates
- Deserialize regex while parsing
- Rename group_parsers to commit_parsers
- Remove bare unwrap from capitalize_first_filter
- Rename changelog module to template
- Add changelog module
- Remove `ReleaseRoot` struct
- Add section to configuration file
- Rename capitalize_first filter to upper_first
- Make commit parsers optional
- Make changelog header/footer optional
- Rename prepend argument to changelog
- Add possible values to strip argument
- Make skip_tags optional
- Update the order and value names
- Update the value name for `--strip`
- Improve logging
- Update value names and description

### Styling

- Update the formatting
- Update formatting
- Update formatting
- Update structure

### Testing

- Update tests
- Add tests
- Add integration tests
- Add tests
```